Swagger Error: Swagger opens up at port 80 successfully.

But when I give swagger.json inthat path as: 
https://github.com/amodankulkar/OperationalizingML/blob/master/starter_files/swagger/swagger.json

It throws an error:

Failed to load API definition.
Errors
Hide
 
Fetch errorFailed to fetch https://github.com/amodankulkar/OperationalizingML/blob/master/starter_files/swagger/swagger.json
Fetch errorPossible cross-origin (CORS) issue? The URL origin (https://github.com) does not match the page (http://localhost). Check the server returns the correct 'Access-Control-Allow-*' headers.

This is expected as the program serve.py did not run successfully.
It also gave me the error as:
python serve.py 8000
Traceback (most recent call last):
  File "serve.py", line 40, in <module>
    test(CORSRequestHandler, HTTPServer, port=port)
  File "C:\Users\sinamo\miniconda3\lib\http\server.py", line 1230, in test
    with ServerClass(server_address, HandlerClass) as httpd:
  File "C:\Users\sinamo\miniconda3\lib\socketserver.py", line 449, in __init__
    self.server_bind()
  File "C:\Users\sinamo\miniconda3\lib\http\server.py", line 137, in server_bind
    socketserver.TCPServer.server_bind(self)
  File "C:\Users\sinamo\miniconda3\lib\socketserver.py", line 463, in server_bind
    self.socket.bind(self.server_address)
TypeError: an integer is required (got type str)